dts/bindings/ieee802154/ti,cc1200.yaml

```diff
@@ -6,3 +6,8 @@ description: Texas Instruments CC1200 802.15.4 wireless transceiver
 compatible: "ti,cc1200"
 
 include: spi-device.yaml
+
+properties:
+    int-gpios:
+      type: phandle-array
+      required: true
```

Comment:
The standard practice now (see porting comment) is to document the active level for both input and output signals as perceived by the device.

Reply:
Could you pin-point to an actual example here?